help/md/srGelmanRubin.md

```diff
@@ -15,32 +15,30 @@ The Gelman–Rubin rule compares the variance between chains with the variance w
 
 ## example
 ```
-# Specify a model
-mymodel = model(tree)
+# Binomial example: estimate success probability given 7 successes out of 20 trials
+r ~ dnExp(10)
+p := Probability(ifelse(r < 1, r, 1))
+n <- 20
+k ~ dnBinomial(n, p)
+k.clamp(7)
+mymodel = model(k)
 
-# Define monitors to monitor parameters for convergence
-paramFile = "parameters.log"
-# Configure the parameter file to monitor only the parameters whose convergence shall be assessed
-monitors[1] = mnModel( filename = paramFile, printgen = 10, stochasticOnly = TRUE, exclude = ["rel_br_lengths"] )
+moves = VectorMoves()
+moves.append( mvSlide(r, delta=0.1, weight=1) )
 
-# Define threshold rules (optional): Stop as soon as ANY has been met
-stopping_rules[1] = srMaxIteration(200000)
-stopping_rules[2] = srMaxTime(15, "hours")
+paramFile = "parameters.log"
 
-# Check for convergence every 1000 iterations
-convergenceFreq = 1000
+monitors = VectorMonitors()
+monitors.append( mnModel(filename=paramFile, printgen=100, p) )
 
-# Add convergence rules: Stop as soon as ALL have been met
-stopping_rules[3] = srMinESS(50, file = paramFile, freq = convergenceFreq)
-stopping_rules[4] = srGelmanRubin(1.01, file = paramFile, freq = convergenceFreq)
-stopping_rules[5] = srGeweke(prob = 0.001, file = paramFile, freq = convergenceFreq)
-stopping_rules[6] = srStationarity(prob = 0.01, file= paramFile ,freq = convergenceFreq)
+# Stop when the potential scale reduction factor falls below 1.01
+stopping_rules[1] = srGelmanRubin(1.01, file = paramFile, freq = 1000)
 
+# Create the MCMC object
 mymcmc = mcmc(mymodel, monitors, moves)
```

Comment:
We should add `nruns=2` here, because the Gelman-Rubin statistic is only defined for 2 or more chains.